modulefiles/wcoss_cray/fv3

```diff
@@ -65,7 +65,7 @@ module swap pmi pmi/5.0.11
 ##
 ## load cmake
 ##
-module load cmake/3.6.2
+module load cmake/3.16.3
```

Comment:
Hi @BijuThomas-NOAA ,

Based on the conversations in [this issue](https://github.com/ufs-community/ufs-weather-model/issues/62), could you please change back to load cmake/3.6.2 but add a module use on top of it like:
module use /usrx/local/dev/modulefiles
module load cmake/3.6.2

It's also better to remove the redundant module load for cmake above as well.

Thanks!

Bin